extras/asr-services/init.py

```
@@ -395,38 +499,57 @@ def update_config_yml(self, provider: str):
                         else:
-                            self.console.print(f"[yellow][WARNING][/yellow] Model '{name}' not found in defaults.yml either")
+                            self.console.print(
+                                f"[yellow][WARNING][/yellow] Model '{name}' not found in defaults.yml either"
+                            )
 
                     config["models"] = models
                     config_manager.save_full_config(config)
                 else:
-                    self.console.print(f"[yellow][WARNING][/yellow] defaults.yml not found, cannot add missing models")
+                    self.console.print(
+                        f"[yellow][WARNING][/yellow] defaults.yml not found, cannot add missing models"
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)